test-framework/junit5/src/main/java/io/quarkus/test/junit/QuarkusMainIntegrationTestExtension.java

```diff
@@ -167,8 +167,12 @@ private ArtifactLauncher.LaunchResult doProcessStart(ExtensionContext context, S
                         System.setProperty(i.getKey(), i.getValue());
                     }
                 }
-                if (testResourceManager != null) {
-                    testResourceManager.close();
+                try {
+                    if (testResourceManager != null) {
+                        testResourceManager.close();
+                    }
+                } catch (Exception e) {
+                    System.err.println("Unable to shutdown resource: " + e.getMessage());
```

Comment:
Is it too early/late to use the logging mechanism here?

Reply:
I have never found it working reliably